app/views/set_ups/index.html.erb

```diff
@@ -1,2 +1,12 @@
 <h1>SetUps#index</h1>
 <p>Find me in app/views/set_ups/index.html.erb</p>
+
+<% @set_ups.each do |set_up| %>
+  <%= link_to "link", set_up_path(set_up) %>
+  <%= set_up.start_date %>
+  <%= set_up.end_date %>
+  <%= ((set_up.health_checks.count) / (set_up.company.users.count).to_f * 100).ceil %>%
```

Comment:
Some users/employees have several health checks at the same date. Please check the seed.